src/tests/ftest/pool/bad_query.yaml

```diff
@@ -29,3 +32,11 @@ querytests:
           info:
              - NULL
              - PASS
+   connectmode:
+     mode: 2
+   createmode:
+     mode: 511
+   createset:
+     setname: daos_server
+   createsize:
+     size: 1073741824
```

Comment:
Why did you add these? They are defined in L11-15 and passed in to `TestPool` when we call `self.pool.get_params(self)`

Reply:
bad_query.py is parsing and using the classical parameters.

Reply:
Where is it parsing?

Reply:
@shimizukko is correct, these added yaml parameters are not used by the test.

Reply:
You are right, bad_query is using the new method in TestPool.
bad_evict is parsing the old parameters.